addons/mail/static/tests/thread/thread.test.js

```diff
@@ -165,20 +165,6 @@ test("display day separator before first message of the day", async () => {
     await contains(".o-mail-DateSection");
 });
 
-test("do not display day separator if all messages of the day are empty", async () => {
-    const pyEnv = await startServer();
-    const channelId = pyEnv["discuss.channel"].create({ name: "" });
-    pyEnv["mail.message"].create({
-        body: "",
-        model: "discuss.channel",
-        res_id: channelId,
-    });
-    await start();
-    await openDiscuss(channelId);
-    await contains(".o-mail-Thread", { text: "The conversation is empty." });
-    await contains(".o-mail-DateSection", { count: 0 });
-});
```

Comment:
instead of removing these tests, we should inverse them to assert what is supposed to be shown

Reply:
More tests means more code maintenance. I'm not against test when they are relevant, such as the code had tweaks and there's test coverage to give confidence this stays.

However here when a message is deleted, it keeps it in place and shows "This message has been removed". There's no longer the need for the tweak of new message separator placement. I don't see a point to maintain a test that test something that works on any other message. There's no code subtleties to manage these list of all deleted messages in a single day.